roles/cloud_devops_engineer.md

```diff
@@ -0,0 +1,91 @@
+# Cloud / DevOps Engineer
+
+Bristol, UK.
+
+Please apply for this role at [www.madetech.com/careers](https://www.madetech.com/careers).
+
+Our Senior Cloud Engineers deliver digital, data and technology outcomes that improve society. They do this by architecting and delivering cloud solutions, and coaching others to do so both within Made Tech and in public sector organisations.
+
+In Bristol, we are looking for Senior Cloud Engineers with a love for cloud, automation, slick tooling, and empowering software delivery through coaching teams. While we are looking for cloud engineers that are able to demonstrate significant expertise in the design and execution of cloud-based solutions, we expect all of our engineers to have a solid grasp of software engineering fundamentals.
```

Comment:
Remove "In Bristol" given HMRC's policies we can look wider.

Reply:
Should I remove the "Bristol, UK." from line 3 then? Or add other regions in? Talent will create a workable job entry per region.